### PR TITLE
Update Cargo.toml files to reference new repo name

### LINKED ISSUE #403: Update Cargo.toml files to reference new repo name
Replace "github.com/CosmWasm/cosmwasm-plus" references with "github.com/CosmWasm/cw-plus"

### PR DESCRIPTION
**Pull request by ueco-jb**
closes #403

### CODE REVIEW

**Comment by ueco-jb** [comment]
@ethanfrey there are still several places `.md` (mostly readmes), should they be replaced as well?

**Comment by ethanfrey** [comment]
Thanks for taking this on.

Yes, please update links and names in MD files as well.

no cosmwasm-plus should be spared from renaming

File: CONTRACTS.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ the interfaces defined in `packages/cw*`.
 ## Creating a new contract
 
 Use [`cosmwasm-template`](https://github.com/CosmWasm/cosmwasm-template) as a
-basis, in particular the `cosmwasm-plus` branch.
+basis, in particular the `cw-plus` branch.
```
Comment (ethanfrey) [inline CONTRACTS.md:9]: ah I need to rename that branch as well. can you make an issue on cw-template so it doesn't get forgotten

**Review by ethanfrey — APPROVED** [review]
looks good. please create another issue before/after merge

File: CONTRACTS.md (hunk 1 above)
Comment (ueco-jb) [reply CONTRACTS.md:9]: Done https://github.com/CosmWasm/cosmwasm-template/issues/83

**Pull request merged**